### PR TITLE
Update function visibility example

### PR DESCRIPTION
**Pull request by kronosapiens**
Two functions don't access state and should be `pure`. Also, inconsistent spacing when using arithmetic.

Let me know what you think!

### CODE REVIEW

File: docs/contracts.rst [hunk 1]
```diff
@@ -194,10 +194,10 @@ In the following example, ``D``, can call ``c.getData()`` to retrieve the value
     contract C {
         uint private data;
 
-        function f(uint a) private returns(uint b) { return a + 1; }
+        function f(uint a) private pure returns(uint b) { return a + 1; }
         function setData(uint a) public { data = a; }
         function getData() public returns(uint) { return data; }
```
Comment (axic) [inline docs/contracts.rst]: This should be `view` if we are upgrading this non-compiling example :)

Comment (kronosapiens) [reply docs/contracts.rst]: Why? It only uses passed arguments. It doesn't read contract state.

Comment (kronosapiens) [reply docs/contracts.rst]: Also, can you help me understand the build failure message? I'm reading the output but I don't see the error. It looks like it's some sort of file-not-found within the build machine.

```
WARNING: The system cannot find the path specified.
C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Redist\MSVC\14.13.26020\x86\Microsoft.VC141.CRT\
2 files, 27103744 bytes (26 MiB)
```

Comment (axic) [reply docs/contracts.rst]: You haven't changed `getData()` and it is lacking a visibility specifier, but should be `view`. The others you have changed to `pure` makes sense.

I am not sure about the error message unfortunately.

Comment (kronosapiens) [reply docs/contracts.rst]: Ah I see. I can update `getData` as well.

Comment (chriseth) [reply docs/contracts.rst]: @kronosapiens the error message you are getting is a known issue.

**Review by chriseth — APPROVED** [review]

**Pull request merged**